.github/workflows/tpl-deploy-app.yml

```diff
@@ -160,6 +164,7 @@ env:
   TESTNET_RPC_URL: "${{ secrets.TESTNET_RPC_URL }}"
   ETHERSCAN_API_KEY: "${{ secrets.ETHERSCAN_API_KEY }}"
   VITE_ALCHEMY_ID: "${{ secrets.VITE_ALCHEMY_ID }}"
+  IPFS_GATEWAYS: "${{ inputs.ipfs-gateways }}"
```

Comment:
Forgot to bump helm chart version perhaps to use this env var?

Reply:
This variable is in helm chart already, but till now it was using default values.

From values.yaml:
```
webClient:
[....]
  ipfsGateways: "https://turquoise-accused-gayal-88.mypinata.cloud/ipfs/,https://octant.infura-ipfs.io/ipfs/" # Note the s (plural form)
```